src/assets/global.scss

```diff
@@ -1,4 +1,5 @@
 @import './variables';
+@import './raleway';
```

Comment:
Are you sure [Raleway](https://www.google.com/fonts/specimen/Raleway) is the "main" Materialize font? It doesn't look Material Design at all... 
If I go to the Materialize website and inspect any random `<h2>` tag, the font I find is rather [Roboto](https://www.google.com/fonts/specimen/Roboto) which, to me, looks much more Material Design than Raleway.